README.md

```diff
@@ -96,6 +96,33 @@ When a PR passes the above check, `jira-lint` will also add the issue details to
  </figcaption>
 </figure>
 
+#### Issue Status Validation
+Issue status is shown in the [Description](#description).
+**Why validation issue status?** 
+In some cases, you may be pushing changes for a story that is set to `Done` or it may not have been pulled into working backlog. This section describes how we can help engineers avoid pushing to stories that are set to invalid statuses.
```

Comment:
```suggestion
In some cases, you may be pushing changes for a story that is set to `Done` or it may not have been pulled into working backlog. This section describes how we can help engineers avoid pushing to branches for stories that are set to statuses other than the ones allowed in the project; for example - you may want to allow PRs for stories that are in `To Do`/`Planning`/`In Progress` states.
```